### ISSUE TITLE
One date not showing

### ISSUE THREAD

**Issue by yzpmpgq** [kalenjordan/custom-reports#31]
First, LOVE the extension and it's probably not an issue, but user error/misunderstanding.  I have a daily sales query that has worked great until we had a very large day (7/31).  It just leaves a blank on the graph.  The query works fine, as it shows the amount for the day.  I didn't see any parameter that would restrict the output, so am I missing something?
![report_graph](https://cloud.githubusercontent.com/assets/13834937/9307383/eee21322-44c3-11e5-8cf8-c76f098ed775.jpg)
![report_result](https://cloud.githubusercontent.com/assets/13834937/9307384/eee4fca4-44c3-11e5-9243-5b0325040314.jpg)
![report_sql](https://cloud.githubusercontent.com/assets/13834937/9307385/eeea001e-44c3-11e5-8385-22fd7bc02244.jpg)

**Comment by kalenjordan** [comment]
Thanks!  Hm, interesting - I'm not sure off the top.  Can you try running the exact query that's generating the graph and see if it's missing anything?  Maybe that's what your second screenshot is?

If that's not the issue, then I would wonder if it's an edge case in the graph library for July 31.  Maybe it thinks July only has 30 days or something crazy like that.  Maybe updating the query so that it subtracts a day from the date and see if it renders the same.

**Comment by yzpmpgq** [comment]
Yes, the second screenshot was the result of the query.  It's getting an
amount, just not displaying on the graph.  I tried pulling and moving back
1 day.  Now the output shows 7/30 as the large day, but the graph is again
blank on that day.

**Comment by kalenjordan** [comment]
So the graph is blank on 7/30 and not on 7/31 when you subtract everything
by a day?

On Mon, Aug 24, 2015 at 11:48 AM, yzpmpgq <notifications@github.com> wrote:

> Yes, the second screenshot was the result of the query. It's getting an
> amount, just not displaying on the graph. I tried pulling and moving back
> 1 day. Now the output shows 7/30 as the large day, but the graph is again
> blank on that day.
>
> On Mon, Aug 24, 2015 at 11:45 AM, kalenjordan <notifications@github.com>
> wrote:
>
> > Thanks! Hm, interesting - I'm not sure off the top. Can you try running
> > the exact query that's generating the graph and see if it's missing
> > anything? Maybe that's what your second screenshot is?
> >
> > If that's not the issue, then I would wonder if it's an edge case in the
> > graph library for July 31. Maybe it thinks July only has 30 days or
> > something crazy like that. Maybe updating the query so that it subtracts
> a
> > day from the date and see if it renders the same.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/kalenjordan/custom-reports/issues/31#issuecomment-134295498
> >
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalenjordan/custom-reports/issues/31#issuecomment-134335498>
> .
>



-- 
Kalen Jordan
Founder, MageMail

Listen to MageTalk - The Magento Podcast <http://magetalk.com/> | MageMail
- The Triggered Email App for Magento
<http://magemail.co/?utm_medium=email&utm_source=signature&utm_campaign=signature>

**Comment by yzpmpgq** [comment]
correct

**Comment by kalenjordan** [comment]
Hm - I wonder if the library has some kind of an issue with numbers over a certain thresshold.  Maybe try subtracting something from the value in the query and see if it graphs it.

Sorry I know this isn't very insightful troubleshooting.  Wish I could dedicate more time to supporting this project!

**Comment by yzpmpgq** [comment]
So I figured out if the amount is over $1000, it doesn't show.  Is that a
setting or something to do with the google charting you think?
